<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab7</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>561</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextEdit" name="textEdit"/>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget"/>
       </item>
       <item>
        <widget class="QListView" name="listView"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Enter a value to add and choose operation</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="enterValue">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>80</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxLR">
         <property name="text">
          <string>Output left to right</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="addR">
         <property name="text">
          <string>Add element to the right</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addL">
         <property name="text">
          <string>Add element to the left</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delR">
         <property name="text">
          <string>Delete element from right</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delL">
         <property name="text">
          <string>Delete element from left</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="timesScalarButton">
         <property name="text">
          <string>Times scalar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonMeanValue">
         <property name="text">
          <string>Mean value</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="outIndex">
         <property name="text">
          <string>Display element by index</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonBubbleSort">
         <property name="text">
          <string>Bubble sort</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonSelectionSort">
         <property name="text">
          <string>Selection sort</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxOutputToTextEdit">
         <property name="text">
          <string>Output to TextEdit</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxOutputToTableWidget">
         <property name="text">
          <string>Output to TableWidget</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxOutputToListView">
         <property name="text">
          <string>Output to ListView</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonInputFromTextEdit">
         <property name="text">
          <string>Input from TextEdit</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonInputFromTableWidget">
         <property name="text">
          <string>Input from TableWidget</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
